package.json

```diff
@@ -208,7 +208,7 @@
   "license": "GPL-3.0-or-later",
   "dependencies": {
     "@amplitude/analytics-browser": "^1.1.4",
-    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#main-artifacts",
+    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#0.0.0-rc4-artifacts",
```

Comment:
why aren't we making NPM packages and versioning them?

Reply:
We (almost) don't use anything that is JS related, so we wanted to experiment with different release techniques.
If needed we can create a JS package, but it feels somewhat wrong to use NPM to just pin some non-js files to a specific version.